@here/harp-mapview/lib/MapView.ts

```diff
@@ -2036,6 +2069,43 @@ export class MapView extends THREE.EventDispatcher {
         }
     }
 
+    /**
+     * Set camera, so all target points (or [[GeoBox]]) are visible.
+     *
+     * If not specified current `tilt` and `heading` is used.
+     *
+     * Note.
+     *
+     * This methods uses best effort to find best view for given set of points without a priori
+     * knowledge of any relationship between points (like polygon ordering) or deciding which side
+     * of polygon shall be visible, so it doesn't support well inputs that have longitude span
+     * significantly bigger than 180.
+     *
+     * In particular, if you give set of points on both sides of antimeridian, with flat projection
+     * and `tileWrappingEnabled` feature it will use assume that map is wrapped around and will
+     * find "minimal" box in projected/wrapped world instead showing whole world.
+     *
+     * Options
+     *  - `margin` - margin added to view, `0` - no margin, `1` - 100% of original size, defaults to
+     *     `0.1` meaning 10% is added to calculated distance, so approximately 10% margin is visible
+     *  - `tilt`   - optional, change tilt, defaults to current [[MapView]] tilt
+     *  - `heading` - optional, change heading, defaults to current [[MapView]] heading
+     *
+     * @param target [[GeoBox]] or list of points conforming to [[GeoCoordLike]]
+     */
+    fitBounds(target: GeoCoordLike[] | GeoBox, options?: FitBoundsOptions) {
```

Comment:
can we merge this method into the lookAt method? How are competitors doing this?

Reply:
@ninok Yes, we can merge it. Actually it's perfect fit for overload of `lookAt` if target is set of geocoords of GeoBox.

I would leave `fitBounds` as separate API and add overload for `lookAt`.